packages/storefront-lwc/src/modules/commerce/productLineItem/productLineItem.js

```diff
@@ -14,42 +14,8 @@ import { ShoppingBasket } from 'commerce/data';
 export default class ProductLineItem extends LightningElement {
     @api product;
 
-    /**
-     * Gets item Total After Discount
-     */
-    get itemTotalAfterDiscount() {
-        if (this.product && this.product.itemTotalAfterDiscount) {
-            return this.product.itemTotalAfterDiscount;
-        }
-        return null;
-    }
-
-    /**
-     * Gets the item Total Not Adjusted
-     */
-    get itemTotalNonAdjusted() {
-        if (
-            this.product &&
-            this.product.itemTotalNonAdjusted &&
-            this.product.itemTotalAfterDiscount
-        ) {
-            return this.product.itemTotalAfterDiscount !==
-                this.product.itemTotalNonAdjusted
-                ? this.product.itemTotalNonAdjusted
-                : null;
-        }
-        return null;
-    }
-
     removeHandler(event) {
         const itemId = event.srcElement.getAttribute('data-itemid');
-        ShoppingBasket.removeItemFromBasket(itemId)
-            .then(basket => {
-                this.basket = basket;
-                this.products = basket.products ? basket.products : [];
-            })
-            .catch(error => {
-                console.error('Error received in removing product', error);
-            });
+        ShoppingBasket.removeItemFromBasket(itemId);
```

Comment:
.then and .catch were not doing anything and were not needed at all. So cleaned it up.